webserver/src/main/java/io/kestra/webserver/controllers/NamespaceFileController.java

```diff
@@ -49,6 +49,10 @@ public class NamespaceFileController {
         }
     }
 
+    private final List<Pattern> forbiddenPathPatterns = List.of(
+        Pattern.compile("/_flows.*")
```

Comment:
```suggestion
        Pattern.compile("^/_flows.*")
```

I think it's more correct to add `^` to denote the beginning of the line

Reply:
ups forgot, you're right

Reply:
nvm as I use "matches" method, it automatically prepend with "^" and append with "$" so it's already an entire match regex